website/content/docs/concepts/oidc-provider.mdx

```diff
@@ -0,0 +1,144 @@
+
+The allowed client IDs parameter utilizes the list of client IDs that have been generated by Vault as a part of client registration. By default, all clients will be *disallowed*. Providing an asterisk(*) as the parameter value will allow all clients to use the provider. The scopes parameter employs a list of references to named scope resources. The values provided are discoverable by the `scopes_supported` key in the OIDC discovery document of the provider. By default, a provider will have the `openid` scope available. See the scopes section below for more details on the `openid` scope.
+
+### Scopes
+
+Providers may reference scope resources via the `scopes_supported` parameter to make specific identity information available as claims.
+
+A scope will have the following configuration parameters:
+
+* **Description**: identity information captured by the scope
+* **Template**: maps individual claims to Vault identity information
+
+
+The template parameter takes advantage of the [JSON-based templating](/api-docs/secret/identity/tokens#template) used by identity tokens for claims mapping. This means the parameter will take a JSON string of arbitrary structure where the values may be replaced with specific identity information. Template parameters that are not present for a Vault identity are omitted from the resulting claims without an error.
+
+
+Example of a JSON template for a scope:
+
+```
+{
+  "groups": {{identity.entity.groups.names}},
+  "email": {{identity.entity.metadata.user_email}},
+  "auth0": {
+      "name": {{identity.entity.aliases.$MNT_ACC.metadata.user}}
+  }
+}
+```
+
+The full list of template parameters include:
+
+* `identity.entity.id`
+* `identity.entity.name`
+* `identity.entity.groups.ids`
+* `identity.entity.groups.names`
+* `identity.entity.metadata`
+* `identity.entity.metadata.<metadata key>`
+* `identity.entity.aliases.<mount accessor>.id`
+* `identity.entity.aliases.<mount accessor>.name`
+* `identity.entity.aliases.<mount accessor>.metadata`
+* `identity.entity.aliases.<mount accessor>.metadata.<metadata key>`
+* `time.now`
+* `time.now.plus.<duration>`
+* `time.now.minus.<duration>`
+
+
+Several named scopes can be made available on an individual provider. Note that the top-level keys in a JSON template may conflict with those in another scope. When scopes are made available on a provider, their templates are checked for top-level conflicts. An error will occur if any conflict exists.
```

Comment:
This needs to be clarified a bit. Suggestion below

Reply:
```suggestion
Several named scopes can be made available on an individual provider. Note that the top-level keys in a JSON template may conflict with those in another scope. When scopes are made available on a provider, their templates are checked for top-level conflicts. A warning will be issued to the vault operator if any conflicts are found. Note that this may result in an error if the scopes are requested in an OIDC Authentication Request.
```